src/js/letters/utils/helpers.jsx

```diff
@@ -339,3 +339,37 @@ export function resetDisallowedAddressFields(address) {
 
   return newAddress;
 }
+
+/**
+ * Traverses a single-level object and removes its zero-length own-enumerable properties
+ * @param {Object} input an object with no nested properties
+ * @returns a cloned object with no empty properties
+ */
+export const stripEmpties = (input) => {
+  const newObject = { ...input };
+  const deleteProperty = (key) => (delete newObject[key]);
```

Comment:
Out of idle curiosity, why split this out here instead of defining the function directly in the `.forEach()` call?